CONTRIBUTING.md

```diff
@@ -3,21 +3,21 @@
 Thank you for your interest in contributing to our project. Whether it's a bug report, new feature, correction, or additional
 documentation, we greatly value feedback and contributions from our community.
 
-Open Job Description is under active development. We're releasing this because we'd like your feedback and participation 
```

Comment:
Would it be possible to back out the style changes and submit as a separate PR so it can be a sep commit?

Reply:
Sure. I'll add the spaces back in.